lib/server/index.js

```diff
@@ -12,9 +12,11 @@ riot.parsers = compiler.parsers
 // allow to require('some.tag')
 require.extensions['.tag'] = function(module, filename) {
   var src = riot.compile(require('fs').readFileSync(filename, 'utf8'))
+  var preTag = src.substring(0, src.indexOf('riot.tag'))
+  var tagDefinitaion = src.substring(src.indexOf('riot.tag'))
   module._compile(
-    'var riot = require(process.env.RIOT || "riot/riot.js");module.exports =' + src
-  , filename)
+    'var riot = require(process.env.RIOT || "riot/riot.js");' + preTag + ';module.exports =' + tagDefinitaion
```

Comment:
Could you please rename your variable `tagDefinitaion` => `tagDefinition` please?